doc/radosgw/multisite.rst

```diff
@@ -841,7 +841,7 @@ For example:
 
    radosgw-admin realm get --rgw-realm=movies [> filename.json]
 
-The CLI will echo a JSON object with the realm properties.
+The CLI will echo a JSON object that contains the realm properties:
```

Comment:
`realm's`?

Reply:
I find "echo" to be a very strange verb to use here, given that it seems to come from the standard command in shells used to display text on the terminal. But maybe that's standard for these types of docs when referring to a CLI. I wonder if "display" or "list" might work better.

Reply:
Agreed, I mike take out the line altogether.

Reply:
I'm taking this line out altogether, as Anthony suggested.